docs/book/usage/quilt.md

```diff
@@ -46,13 +46,21 @@ can be the target.
 
 ### Lower cost
 
-This is the most clear and significant use case. *quilt* is especially advantageous for
+This is the most clear and significant use case. *Quilt* is especially advantageous for
 managing large volumes of small blobs, as long as they can be grouped together by the
-user. By consolidating multiple small blobs into a single quilt, storage costs can be
-reduced dramatically—up to 50 times for files around 100KB—making it an efficient
-solution for cost-sensitive applications.
+user. The cost savings come from two sources:
 
-The following table demonstrates shows the potential cost savings in WAL when storing 600 small
+- **Walrus storage and write fees**: By consolidating multiple small blobs into a single
+    quilt, storage costs can be reduced dramatically—more than 400x for files around 10KB—making
+    it an efficient solution for cost-sensitive applications.
+
+- **Sui computation and object storage fees**: Storing many blobs as a single quilt
+    significantly reduces Sui gas costs. In our test runs with 600 files stored in a quilt,
+    we observed **238x** savings in Sui fees compared to storing them as individual blobs.
+    Notably, Sui cost savings depend primarily on the number of files per quilt rather than
```

Comment:
Nit: I think the cost savings *only* depend on the number of files.
```suggestion
    Notably, Sui cost savings only depend on the number of files per quilt rather than
```